api/controllers/game/pass.js

```diff
@@ -45,6 +45,7 @@ module.exports = function (req, res) {
       const victory = {
         gameOver: false,
         winner: null,
+        currentMatch: null,
```

Comment:
Doing null above also matches the shape we're going for here so we don't get different schemas based on request.